C/other/UniqueNumberIII.c

```diff
@@ -0,0 +1,52 @@
+#include<stdio.h>
+main()
+{
+    int n;
+    printf("Enter number of elements: ");
+    scanf("%d",&n);
+    int arr[n],i;
+    printf("\nEnter %d elements: ",n);
+    for(i=0;i<n;i++)
+    {
+        scanf("\n%d",&arr[i]);
+    }
+    int one=0;
+    int two=0;
+    int n_t,x;
+    for(i=0;i<n;i++)
+    {
+        x = arr[i];
+        two |=one & x;
+        one ^=x;
+        n_t =~(one & two);
+        one &= n_t;
+        two &= n_t;
+    }
+    printf("The number that occurs only once: %d \n", one);
+}
+
+/*
+Time complexity: O(n)
+Space complexity: O(n)
+
+Example 1:
+Sample input:
+Enter number of elements: 7
+Enter 7 elements: 6 
+2
+5
+2
+2
+6
+6 
+Output: 5
+
+Example 2:
+Sample Input: 
+Enter number of elements: 4
+Enter 7 elements: 3 
+3
+1
+3 
+Output: 1 

```

Comment:
Sample output should be the output returned when the program is run in an IDE.
Something like ```The number that occurs only once is: 1```